api/src/config.rs

```diff
@@ -427,6 +441,14 @@ pub struct S3Config {
     pub mirrors: Vec<MirrorConfig>,
 }
 
+/// ContentProxyProxy configuration information to access blobs.
+#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+pub struct LocalHttpProxyConfig {
```

Comment:
How about `LocalUdsConfig`?
We can define an APIs to retrieve blobs from Unix domain socket, no matter it's containerd content store or something else.

Reply:
Or `ContainerdContentStoreProxy`? @imeoer

Reply:
But this backend driver only support http protocol now. I think `LocalHttpProxy` may be better.

Reply:
It's ok for me if using `LocalHttpProxy`, maybe we can merge the feature with [nydus-backend-proxy](https://github.com/dragonflyoss/image-service/tree/master/contrib/nydus-backend-proxy) (not in this PR)? So that we can use `LocalHttpProxy` to request an HTTP server that can proxy to local blob directory/containerd content store.